src/style.css

```diff
@@ -0,0 +1,85 @@
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+@font-face {
+  font-family: "Avenir";
+  src: url(/fonts/Avenir.ttc);
+  font-weight: normal;
+  font-style: normal;
+}
+@font-face {
+  font-family: "Avenir";
+  src: url(/fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(/fonts/Avenir-Heavy.ttf);
+  font-weight: 500;
+  font-style: normal;
+}
+
```

Comment:
So here you will have a class `.form`, that have common styles for form 

_it's an example!_
```
.form {
  margin: 20px 0;
  border: solid 1px #e1e7ed;
  border-radius: 4px;
  height: 70px;
  position: relative;
  margin-top: 20px;
}
```
And then you use modifier that have ONLY differences with base class form

```
.form--small {
  height: 42px;
  border-radius: 4px;
}
```

So in this way you follow DRY (Don't repeat yourself) principe 
Apply this principe all over the styles

Reply:
Thank you!